plugins/outputs/nebius_cloud_monitoring/sample.conf

```diff
@@ -0,0 +1,10 @@
+# Send aggregated metrics to Nebius.Cloud Monitoring
+[[outputs.nebius_cloud_monitoring]]
+  ## Timeout for HTTP writes.
+  # timeout = "20s"
+
+  ## Nebius.Cloud monitoring API endpoint. Normally should not be changed
+  # endpoint = "https://monitoring.api.il.nebius.cloud/monitoring/v2/data/write"
+
+  ## All user metrics should be sent with "custom" service specified. Normally should not be changed
+  # service = "custom"
```

Comment:
If a user is not suppose to change this, then why is this an option at all? 

What is the impact of a user changing this?

I'm asking because I would hate to get a bug that comes in where someone says they changed this and in reality it is the backend rejecting metrics because of the change.

Reply:
You absolutely right. We need to delete it. It was created for the future development of Nebius Monitoring Backend. Now it stale a bit. So changing of this option will lead to failure.